### PR TITLE
Project 17 Build an API using Express

### PR DESCRIPTION
**Pull request by Angelicabrodin**
I felt that it was really hard to move to back end, again every week goes so fast and you really do not have time to let everything sink in unfortunately. 

I had so much problems with my server, and it was giving a lot of errors, and then I also had problems with heruko, I do not know if this is because I tried changing the port nr so many times to try and see if that was what made my server not working. And in the end, I still have a error when running npm run dev but heruko finally works so that is good. 

I think it is cool when you think about how much you can create, and I wish I had so much more time to experiment, I will hopefully go back to this in the future and try it out a bit more!

### CODE REVIEW

**Comment by JennieDalgren** [comment]
please add link to deployed project

**Comment by Angelicabrodin** [comment]
> please add link to deployed project

@JennieDalgren I thought it was enough with heruko? do I need to deploy it on netlify as well?

File: server.js [hunk 1]
```diff
@@ -2,19 +2,23 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 
+import goldenGlobesData from './data/golden-globes.json'
+import netflixData from './data/netflix-titles.json'
+
+console.log(goldenGlobesData.length)
+console.log(netflixData.length)
```
Comment (JennieDalgren) [inline server.js]: remove this

File: server.js [hunk 2]
```diff
@@ -27,6 +31,63 @@ app.get('/', (req, res) => {
...
+//   "type": "TV Show"
+// },
+
+// Netflix - Titles
+
+app.get('/movies', (req, res) => {
+  res.json(netflixData)
+})
+
+app.get('/releaseyear/:year', (req, res) => {
```
Comment (JennieDalgren) [inline server.js]: again here, call this movies/:year or make it an optional query in the movies endpoint

**Pull request closed without merging**